### ISSUE TITLE
Cannot read properties of undefined (reading 'swc')

### ISSUE THREAD

**Issue by muratgozel** [swc-project/swc-node#792]
after updating
- node from v20.5 to v20.9.0
- @swc-node/register from 1.9.1 to 1.9.2
- @swc/core from 1.5.24 to 1.6.5

i started to get the following error when i run:
```sh
node --env-file ./.env --watch --watch-path src --enable-source-maps --import @swc-node/register/esm-register src/index.ts
```
```log
TypeError [Error]: Cannot read properties of undefined (reading 'swc')
    at load (/Users/muratgozel/workspace/gozel/monolith/node_modules/.pnpm/@swc-node+register@1.9.2_@swc+core@1.6.5_@swc+types@0.1.9_typescript@5.5.2/node_modules/@swc-node/register/esm.mts:72:49)
    at nextLoad (node:internal/modules/esm/hooks:833:28)
    at Hooks.load (node:internal/modules/esm/hooks:416:26)
    at MessagePort.handleMessage (node:internal/modules/esm/worker:168:24)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:807:20)
    at exports.emitMessage (node:internal/per_context/messageport:23:28)
```

**Comment by yeliex** [comment]
https://github.com/swc-project/swc-node/pull/779 should resolved this

**Comment by yeliex** [comment]
duplicate https://github.com/swc-project/swc-node/issues/785

**Comment by yeliex** [comment]
https://github.com/swc-project/swc-node/pull/779 should fixed this

@Brooooooklyn could you check and make a release?

**Comment by muratgozel** [comment]
It's fixed in `@swc-node/register:1.10`
